examples/events/EiffelServiceStartedEvent/simple.json

```diff
@@ -0,0 +1,17 @@
+{
+  "meta": {
+    "type": "EiffelServiceStartedEvent",
+    "version": "1.0.0",
+    "time": 1234567890,
+    "id": "aaaaaaaa-bbbb-cccc-dddd-eeeeeeeeeee5",
+    "source": {
+    "domainId": "example.domain"
+    }
+  },
+    "links": [
+      {
+      "type": "ENVIRONMENT",
+      "target": "aaaaaaaa-bbbb-cccc-dddd-eeeeeeeeeee0"
+      }
```

Comment:
Would this be heading in the right direction ,
    "links": [
      {
      "type": "ENVIRONMENT",
      "target": "aaaaaaaa-bbbb-cccc-dddd-eeeeeeeeeee0"
      "type": "ARTIFACT",
      "target": "aaaaaaaa-bbbb-cccc-dddd-eeeeeeeeeee1"

      }
  ]
}